src/daemon/compilation_daemon.cc

```diff
@@ -343,6 +343,11 @@ base::ProcessPtr CompilationDaemon::CreateProcess(
   if (flags.has_input()) {
     process->AppendArg(Immutable(flags.input()));
   }
+  if (flags.has_asan_blacklist_file()) {
+    auto flag_value = String("-fsanitize-blacklist="_l);
```

Comment:
If possible please use a couple of tokens, like the flags above, i.e. "-fsanitize-blacklist bla-bla-bla".